project/02-Modularity/Handout.md

```diff
@@ -0,0 +1,160 @@
+# Android Project - Part 2: Modularity
+
+In this second part of the SwEng project, you will add weather forecast capabilities to your app. Using an external service, [OpenWeatherMap](https://api.openweathermap.org/), you will display to the user a weather forecast at their location, or at an address of their choice. For that, you will also need to be able to find out where the user is located.
+
+> :information_source: In this part, we will use features that were introduced in Java 8. These features were not available on older versions of Android (Android < N - API Level 24). To use them, we either need to provide alternatives, or to loose compatibility with these older devices. Here, we choose to do the latter. To enable this, please go to your app `build.gradle` and set `minSdkVersion` to `24`.
```

Comment:
nit: use a comma instead of a hyphen within the parentheses to avoid looking like subtraction given the less-than before

loose -> lose